### ISSUE TITLE
 [FreeMotionAnimationLoop] some examples are not working since merge of  #459

### ISSUE THREAD

**Issue by damienmarchal** [sofa-framework/sofa#749]
The example src/applications/plugins/SofaPython/examples/keyboardControl.scn is not working anymore. 

To 'fix' the example it is possible replace 'free_position' by 'position' in the keyboardControl.py pythonScriptController:
```py
       ...
	# key and mouse events; use this to add some user interaction to your scripts 
	def onKeyPressed(self,k):
		
		# free_position is a scalar array : [tx,ty,tz,rx,ry,rz,rw]
		free_position=self.MechanicalState.free_position

		# translation speed
		speed = 1 

		# UP key : front
		if ord(k)==19:
			free_position[0][2]-=speed
		....
	
		self.MechanicalState.free_position=free_position
		return 0 
```

The change of behavior happens with the merge of PR #459 
@ChristianDuriez, @fjourdes, @EulalieCoevoet  is this a regression (and thus we should fix the .cpp) or is this the normal expected behavior (and so we should just update the examples :)) ? 

______________________________________________________

Suggested labels:

**Comment by damienmarchal** [comment]
Hi,

Can anyone give look into this issue ? 
@ChristianDuriez ,@fjourdes, @EulalieCoevoet 

DM.

**Comment by damienmarchal** [comment]
Hi @ChristianDuriez, @fjourdes, @EulalieCoevoet, 

It would be nice to have feedback on this possible issue because this is related to your PR. If it is not an issue we can close it.

**Comment by damienmarchal** [comment]
I finally got the answer from @ChristianDuriez ... the new behavior. (and the good one) is to use position and not free_position. So I will make a small PR.

**Comment by fredroy** [comment]
Fixed in #1020 (not closed automatically I suppose)
